docs/preact-testing-library/api.md

```diff
@@ -0,0 +1,273 @@
+`container.querySelector` etc. to inspect the children.
+
+> Tip: To get the root element of your rendered element, use
+> `container.firstChild`.
+>
+> NOTE: When that root element is a
+> [React Fragment](https://reactjs.org/docs/fragments.html),
+> `container.firstChild` will only get the first child of that Fragment, not the
+> Fragment itself.
+
+> 🚨 If you find yourself using `container` to query for rendered elements then
+> you should reconsider! The other queries are designed to be more resiliant to
+> changes that will be made to the component you're testing. Avoid using
+> `container` to query for elements!
+
+### `baseElement`
+
+The containing DOM node where your React Element is rendered in the container.
+If you don't specify the `baseElement` in the options of `render`, it will
+default to `document.body`.
+
+This is useful when the component you want to test renders something outside the
+container div, e.g. when you want to snapshot test your portal component which
+renders it's HTML directly in the body.
+
+> Note: the queries returned by the `render` looks into baseElement, so you can
+> use queries to test your portal component without the baseElement.
+
+### `debug`
+
+This method is a shortcut for `console.log(prettyDOM(baseElement))`.
+
+```jsx
+import React from 'react'
+import { render } from 'preact-testing-library'
+
+const HelloWorld = () => <h1>Hello World</h1>
+const { debug } = render(<HelloWorld />)
+debug()
+// <div>
+//   <h1>Hello World</h1>
+// </div>
+// you can also pass an element: debug(getByTestId('messages'))
+```
+
+This is a simple wrapper around `prettyDOM` which is also exposed and comes from
+[`dom-testing-library`](https://github.com/kentcdodds/dom-testing-library/blob/master/README.md#prettydom).
+
+### `rerender`
+
+It'd probably be better if you test the component that's doing the prop updating
+to ensure that the props are being updated correctly (see
+[the Guiding Principles section](#guiding-principles)). That said, if you'd
+prefer to update the props of a rendered component in your test, this function
+can be used to update props of the rendered component.
+
+```jsx
+import { render } from 'preact-testing-library'
+
+const { rerender } = render(<NumberDisplay number={1} />)
+
+// re-render the same component with different props
+rerender(<NumberDisplay number={2} />)
+```
+
+[See the examples page](example-update-props.md)
+
+### `unmount`
+
+This will cause the rendered component to be unmounted. This is useful for
+testing what happens when your component is removed from the page (like testing
+that you don't leave event handlers hanging around causing memory leaks).
+
+> This method is a pretty small abstraction over
+> `ReactDOM.unmountComponentAtNode`
+
+```jsx
+import { render } from 'preact-testing-library'
+
+const { container, unmount } = render(<Login />)
+unmount()
+// your component has been unmounted and now: container.innerHTML === ''
+```
+
+### `asFragment`
```

Comment:
Not seeing support for this option in https://github.com/antsmartian/preact-testing-library/blob/master/src/index.js